.gitignore

```diff
@@ -19,3 +19,4 @@ cypress.env.json
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+/tsconfig.tsbuildinfo
```

Comment:
There's a new line in the `.gitignore` is that intentional?

I think this has to do with TypeScript incremental builds?

Reply:
Yeah, it was on purpose. [This file shouldn't be included in VCS](https://betterstack.dev/blog/lerna-typescript-monorepo/).

Reply:
Okay! 😄